moveit2_galactic.repos

```diff
@@ -1,5 +1,9 @@
 # TODO(henningkayser): Remove once ros2_control is released (https://github.com/ros-controls/ros2_control/issues/421)
 repositories:
+  moveit_resources:
+    type: git
+    url: https://github.com/ros-planning/moveit_resources
+    version: ros2
```

Comment:
I don't think you need this here if you put it in the main one.

Reply:
CI takes that into account:

`UPSTREAM_WORKSPACE: moveit2.repos $(f="moveit2_$(sed 's/-.*$//' <<< "${{ matrix.env.IMAGE }}").repos"; test -r $f && echo $f)`

Reply:
Thinking again you are right, I am removing it and see if it still passes.